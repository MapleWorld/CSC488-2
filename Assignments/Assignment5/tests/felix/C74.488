begin
    if 500 >= 500 then
        put "500 >= 500"
    end
    put skip
    if 600 >= 500 then
        put "600 >= 500"
    end
end
