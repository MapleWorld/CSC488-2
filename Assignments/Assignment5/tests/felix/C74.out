500 >= 500
600 >= 500
